server/core/locking/locking.go

```diff
@@ -116,8 +121,13 @@ func (c *Client) List() (map[string]models.ProjectLock, error) {
 	return m, nil
 }
 
+func (c *Client) GetQueueByLock(project models.Project, workspace string) ([]models.ProjectLock, error) {
+	return c.backend.GetQueueByLock(project, workspace)
+}
+
+// TODO monikma extend the tests
```

Comment:
Has this done? I'm guess you were going to verify `DequeueStatus`?

Reply:
No, and yes..